cmd/loadtest/output.go

```diff
@@ -478,7 +479,11 @@ func lightSummary(lts []loadTestSample, startTime, endTime time.Time, rl *rate.L
 
 	log.Info().Time("startTime", startTime).Msg("Start time of loadtest (first transaction sent)")
 	log.Info().Time("endTime", endTime).Msg("End time of loadtest (final transaction mined)")
-	log.Info().Float64("tps", tps).Msg("Overall Requests Per Second")
+	log.Info().Float64("tps", tps).Msg("Successful Requests Per Second")
+	// Only output total rates per second if there are failed transactions and TPS != RPS
+	if tps != rps {
+		log.Info().Float64("rps", rps).Msg("Total Requests Per Second (both successful and unsuccessful transactions)")
```

Comment:
I think we should log an error here, what do you think?

Reply:
+1 I think that is much more intuitive 
![image](https://github.com/user-attachments/assets/90d0db60-c062-4817-9bf5-03c075ec34a4)